app/views/RoomInfoEditView/SwitchContainer.js

```diff
@@ -15,20 +15,25 @@ export default class SwitchContainer extends React.PureComponent {
 		rightLabelSecondary: PropTypes.string,
 		onValueChange: PropTypes.func,
 		theme: PropTypes.string,
-		testID: PropTypes.string
+		testID: PropTypes.string,
+		labelContainerStyle: PropTypes.object,
+		leftLabelStyle: PropTypes.object,
+		children: PropTypes.any
 	}
 
 	render() {
 		const {
-			value, disabled, onValueChange, leftLabelPrimary, leftLabelSecondary, rightLabelPrimary, rightLabelSecondary, theme, testID
+			children, value, disabled, onValueChange, leftLabelPrimary, leftLabelSecondary, rightLabelPrimary, rightLabelSecondary, theme, testID, labelContainerStyle, leftLabelStyle
 		} = this.props;
 		return (
 			[
```

Comment:
Since you're changing this file, can you replace
```
[
  {element},
  {element}
]
```
with
```
<>
  {element}
  {element}
</>
```

?

Reply:
I've refactored this component to a memoized functional one

Reply:
I've refactored the SwitchContainer to be a memoized funcional component.